### PR TITLE
refactor: Crawler/kms keys

### LINKED ISSUE #210: Subtasks: Implement crawl factory for KMS resource.
#153

### PR DESCRIPTION
**Pull request by peb-peb**
## NOTE

**closes**
closes #210 
ref to #153 

@mshudrak @ZetaTwo

### CODE REVIEW

File: src/gcp_scanner/crawler/crawler_factory.py [hunk 1]
```diff
@@ -55,6 +56,7 @@
   "static_ips": ComputeStaticIPsCrawler,
   "subnets": ComputeSubnetsCrawler,
   "sourcerepos": CloudSourceRepoCrawler,
+  "kms": KMSKeysCrawler,
```
Comment (mshudrak) [inline src/gcp_scanner/crawler/crawler_factory.py:59]: minor nit: we try to maintain this list in alphabetical order

**Review by mshudrak — APPROVED** [review]
Please fix minor nit. Other than that LGTM

File: src/gcp_scanner/crawler/crawler_factory.py (hunk 1 above)
Comment (peb-peb) [reply src/gcp_scanner/crawler/crawler_factory.py]: done :+1:

**Comment by peb-peb** [comment]
Conflicts Resolved!

**Pull request merged**